omnibus/package-scripts/agent/preinst

```diff
@@ -96,7 +96,7 @@ if [ "$DISTRIBUTION" != "Darwin" ]; then
             rm -rf $DOWNLOADER_DATA
         fi
 
-        DOWNLOADER_DATA3="$INSTALL_DIR/embedded/lib/python3.7/site-packages/datadog_checks/downloader/data"
+        DOWNLOADER_DATA3="$INSTALL_DIR/embedded/lib/python3.8/site-packages/datadog_checks/downloader/data"
```

Comment:
I think this means that, on CentOS/RHEL/SUSE, when upgrading to 7.17.0 from a lower Agent version, the Python 3.7 downloader data folder won't be removed, because:
- in the (lower) prerm, downloader data removal is not executed on upgrade on these systems, only on uninstall,
- in this (7.17.0) preinst, only the 3.8 downloader data folder is removed.

To fix this specific problem, we should remove here the 3.7 downloader data (since upgrading from 6.13.0-7.16.0 won't remove it on upgrade).

To prevent this from happening in the future, we should remove the 3.8 downloader data folder in the prerm instead, in all cases. That way, the removal of a particular downloader data folder (say 3.X) is done by the scripts of the Agent version using Python 3.X, instead of relying on a future Agent version (which may be using Python 3.Y).to do it when it gets installed.

Reply:
This is actually also done in the `prerm` file, however I didn't thought about the 3.7 dir, I'm adding this.